### ISSUE TITLE
[Bug] API - Portfolio Optmization NCO

### ISSUE THREAD

**Issue by teracapital** [OpenBB-finance/OpenBBTerminal#2283]
running api.portfolio.po.nco
returning:
Error: (il>=1&&il<=n) failed for 6th keyword il: dsyevr:il=1
something related to eigenvalues calculations in riskfolio

Error: not enough values to unpack (expected 2, got 0)
weights, stock_returns = optimizer_model.get_hcp_portfolio

Since calling the function from terminal and calling the same functions and parameters directly through riskfolio works properly it seems to me that the problem is with the default parameters... couldn't find which

**Comment by JerBouma** [comment]
Hi @teracapital,

It is possible to also directly access the model through the API. This is of course not entirely how we intended it to work but it works for now while we are working on correcting this issue. We are currently in the process of greatly improving the API. See the following result:

![image](https://user-images.githubusercontent.com/46355364/183890066-28423765-8df1-42e0-872c-531453a57e52.png) 

@simmonsj330 Could you look into why this returns an error when you work on the `portfolio` refractoring? On my side it seems it is forgetting a dictionary altogether.

![image](https://user-images.githubusercontent.com/46355364/183889487-d5aca791-24d2-47de-9f86-a1b1c0be6d00.png)

**Comment by teracapital** [comment]
@JerBouma one the 1st code, you forgot to set model='NCO', the default model is 'HRP' which is working

on the 2nd code inside optimizer_view.py there is a dictionary that maps argumetns objective and risk_measure from its value to the value used by riskfolio from the api this values should be lowercase like (minrisk and mv) which got mapped to MinRisk and MV afterwards (if you call this with captilized charcters it got mapped to none and brokes the code)

Many other default values in the api is using captlized charecters, just substitute that with the proper value when using the api for now.

**Comment by JerBouma** [comment]
@teracapital You are correct, will ask an Engineer to fix these bugs while the API is being refractored.

**Comment by colin99d** [comment]
<img width="552" alt="Screen Shot 2022-10-17 at 1 06 37 PM" src="https://user-images.githubusercontent.com/72827203/196162067-d44e4207-17c6-4262-8cb2-6963ec4928ba.png">
<img width="592" alt="Screen Shot 2022-10-17 at 1 06 53 PM" src="https://user-images.githubusercontent.com/72827203/196162113-455f6323-e412-4ece-ac75-e70a9a0a35e3.png">
